scripts/build-website.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e # exit on error
+
+
+echo "building via webpack"
+
+yarn webpack --config website/webpack.config.js
+
+echo "building via parcel"
+
+(
+  cd examples/hooks-e-commerce;
+  yarn build --dist-dir ../../website/examples/hooks-e-commerce;
```

Comment:
This will fix the Netlify deployment issue @francoischalifour mentioned in his review.

```suggestion
  yarn build --public-url /examples/hooks-e-commerce --dist-dir ../../website/examples/hooks-e-commerce;
```

Reply:
thanks, that's what I was about to add as well!